cmd/utils/flags.go

```diff
@@ -1122,6 +1129,13 @@ func setBootstrapNodesV5(ctx *cli.Context, cfg *p2p.Config) {
 		urls = SplitAndTrim(ctx.String(BootnodesFlag.Name))
 	case cfg.BootstrapNodesV5 != nil:
 		return // already set, don't apply defaults.
+	case ctx.IsSet(OPNetworkFlag.Name):
```

Comment:
If running with a previous datadir, then this flag isn't explicitly set.

Reply:
That's true, but it should be set for most node operators. I'm not super sure how to get the network if this flag is not set though.

Reply:
Maybe we should just add to the documentation that the network flag is required for snap sync to work?